packages/arb-bridge-eth/contracts/libraries/CloneFactory.sol

```diff
@@ -62,6 +62,19 @@ contract CloneFactory {
         }
     }
 
+    function calculateCreate2CloneAddress(ICloneable target, bytes32 salt) internal view returns (address calculatedAddress) {
+        calculatedAddress = address(uint(keccak256(abi.encodePacked(
+            byte(0xff),
+            address(this),
+            salt,
+            abi.encodePacked(
```

Comment:
The contract bytecode needs to be hashed before encoded with the other fields according to the EIP 

https://eips.ethereum.org/EIPS/eip-1014